tests/wpunit/Tribe/Utils/Date_I18n_ImmutableTest.php

```diff
@@ -34,9 +34,8 @@ public function data_dates_and_timezones() {
 	 */
 	public function it_should_properly_include_timezones( $datetime, $timezone, $expected ) {
```

Comment:
I know this test pre-dates the change you made in this PR, but I'll comment anyway: coming into this raw (I haven't really looked at this body of work before) I took a moment before I grokked what we're testing for (that we expect the result in UTC). The name of the test doesn't _quite_ express that for me. 

Anyway, that's a bit of a subjective comment perhaps.

Reply:
That is totally a worthwhile mention. I've updated the tests to be named: `it_should_properly_convert_to_utc_from_provided_timezone`